charts/mssql/values.yaml

```diff
@@ -0,0 +1,2 @@
+image: mcr.microsoft.com/mssql/server:2022-latest
+mssql_sa_password: X#BcoDVsH^u9tp
```

Comment:
Lets add a comment here explaining that this password is hardcoded for development use and that this helm chart should never be used outside of a localized development environment.